actor/todo-sql/src/lib.rs

```diff
@@ -0,0 +1,261 @@
+use serde::{Deserialize, Serialize};
+use wasmbus_rpc::actor::prelude::*;
+use wasmcloud_interface_httpserver::{HttpRequest, HttpResponse, HttpServer, HttpServerReceiver};
+use wasmcloud_interface_numbergen::generate_guid;
+use wasmcloud_interface_sqldb::{
+    SqlDb, SqlDbError, SqlDbSender,
+    //FetchResult, ExecuteResult,
+    minicbor,
+};
+use wasmcloud_interface_logging::{debug, info, warn};
+use once_cell::sync::Lazy;
+use minicbor::{decode,Decode};
+
+type Db = SqlDbSender<WasmHost>;
+const TABLE_NAME :&str = "todos";
+/// limit characters to avoid sql injection attacks
+/// - english letters uppercase & lowercase
+/// - digits 0-9
+/// - space, comma, period, underscore, dash, slash
+/// that's it!
+static REGEX: Lazy<regex::Regex> = Lazy::new(|| {
+    let re = regex::Regex::new(r"^[a-zA-Z0-9 ,._-/]+$").unwrap();
+    re
+});
+
+#[derive(Serialize, Deserialize)]
+struct InputTodo {
+    title: String,
+    order: Option<i32>,
+}
+#[derive(Serialize, Deserialize)]
+struct UpdateTodo {
+    title: Option<String>,
+    completed: Option<bool>,
+    order: Option<i32>,
+}
+// because 'order' is a keyword in sql, we've renamed the field 'priority',
+// but for serde serialization, leave it as 'order' so that it works
+// with existing javascript written for this app
+#[derive(Serialize, Deserialize, Decode)]
```

Comment:
I think we should comment this type a bit more to be explicit about its use and purpose. For example, I thought this type was being reused for both HTTP interface and SQLDB interface (an anti-pattern) and didn't realize until scrolling down a bit that there are other types being used for the HTTP interface operations.

Reply:
We could be even more explicit and self-documenting and rename the type to `DbTodo` ?

Reply:
My initial goal was to do a port with the fewest number of lines changed so that people could compare the keyvalue and sql implementations. I left the data structures in place that are used by the keyvalue example. A different goal for the todo-sql example could have been to create an opionated model for how to build a todo app, but that was not a goal. I could add some comments to clarify.

Reply:
After starting to write the comment I decided it's better to model best practices than to have the fewest lines of code. Making a DbTodo